### ISSUE TITLE
[Bug] KDBX 4 + argon2id : "InvalidArg: bad KDF algo"

### ISSUE THREAD

**Issue by KaKi87** [keeweb/keeweb#1697]
**Describe the bug**
It is not possible to create an argon2**i**d-hashed KDBX v4 database, although argon2d works.

**To Reproduce**
From homepage, with devtools open :
1. Click *New* (center)
2. Click *New* (bottom left)
3. Scroll down to *key derivation function*
4. Select *argon2id*
5. See error thrown in devtools
<br>*Optionally :*
6. Scroll up to *master password*
7. Type master password
8. Type master password confirmation
9. Click *Lock* (bottom right)
10. Unlock with master password
11. Click *New* (bottom left)
12. Scroll down to *key derivation function*
13. See *argon2d* selected

**Expected behavior**
argon2id works.

**Actual behavior**
The following error is thrown :
```json
{
    "name": "KdbxError",
    "code": "InvalidArg",
    "message": "Error InvalidArg: bad KDF algo"
}
```
And re-opening the databse file will show argon2d instead of argon2**i**d as selected algorithm.

**Screenshots**
GIF :
![](https://user-images.githubusercontent.com/21284089/106460014-fc8c7b80-6492-11eb-978b-6a5c8a471082.gif)

**Environment**
```
KeeWeb v1.16.7 (42b5b00, 2020-12-31)
Environment: electron v11.0.3
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) KeeWeb/1.16.7 Chrome/87.0.4280.67 Electron/11.0.3 Safari/537.36
```

**Kdbx File**
It happens on both demo and new database.

**Logs**
Cf. *actual behavior*

---

Thanks

**Comment by antelle** [comment]
fixed, will be released in v1.17

**Comment by KaKi87** [comment]
Thanks !
